eng/GenAPI.exclusions.txt

```diff
@@ -2,4 +2,11 @@
 T:Microsoft.AspNetCore.Components.RenderTree.RenderTreeFrame
 # Manually implemented - https://github.com/dotnet/arcade/issues/2066
 T:Microsoft.AspNetCore.Mvc.ApplicationModels.PageParameterModel
-T:Microsoft.AspNetCore.Mvc.ApplicationModels.PagePropertyModel
\ No newline at end of file
+T:Microsoft.AspNetCore.Mvc.ApplicationModels.PagePropertyModel
+# Manually implemented - Need to include internal setter
```

Comment:
TODO: Followup

Maybe there's a way to do this without needing to explicitly opt out. Will follow up with experts but for now this works.